articles/cognitive-services/LUIS/luis-container-howto.md

```diff
@@ -67,8 +67,6 @@ Use the [`docker pull`](https://docs.docker.com/engine/reference/commandline/pul
 docker pull mcr.microsoft.com/azure-cognitive-services/luis:latest
 ```
 
-Use the [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) command to download a container image.
```

Comment:
Why would you remove this, this should not be removed.

Reply:
@IEvangelist - It's a duplicate of the line a few lines above it.

Reply:
Ah, ok... thank you for the clarification. You are correct.